GVFS/GVFS.Platform.Mac/GVFS.Platform.Mac.csproj

```diff
@@ -3,14 +3,18 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1;netstandard2.0</TargetFrameworks>
     <Platforms>x64</Platforms>
+    <Version>$(GVFSVersion)</Version>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
+
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <Version>$(GVFSVersion)</Version>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Version)' == ''">

```

Comment:
Why is this new PropertyGroup here? Also, why does the 'Release|x64' property group not have `<Version>$(GVFSVersion)</Version>` in it anymore?

This pattern works fine in other .csproj files in our codebase, is there any reason why we have to make these changes only here?

Reply:
The underlying issue change is addressing is that `dotnet build` and `dotnet test` commands fail because the `Version`  and `AllowUnsafeBlocks` properties are not set for the default configuration those commands run with. This change results in those properties being set regardless of the build configuration.

Reply:
Oh, I see now. That makes sense for your workflow here then!